packages/dev/core/src/Cameras/Inputs/BaseCameraPointersInput.ts

```diff
@@ -85,8 +85,8 @@ export abstract class BaseCameraPointersInput implements ICameraInput<Camera> {
             this._buttonsPressed = evt.buttons;
 
             if (engine.isPointerLock) {
-                const offsetX = evt.movementX || evt.mozMovementX || evt.webkitMovementX || evt.msMovementX || 0;
-                const offsetY = evt.movementY || evt.mozMovementY || evt.webkitMovementY || evt.msMovementY || 0;
+                const offsetX = evt.movementX || 0;

```

Comment:
Why do you need the `|| 0`?

Reply:
TBH, since we're assigning a value in the WDIS, we actually don't need it anymore.  I'll remove it.